pages/common/vault.md

```diff
@@ -0,0 +1,27 @@
+# vault
+
+> A CLI to interact with HashiCorp Vault.
+
+- Create a new vault, requiring at least 2 out of 5 keyshares to unseal:
+
+`vault init -key-shares={{5}} -key-threshold={{2}}`
+
+- Unseal the vault by providing one of the keyshares. Repeat with necessary key-shares until unsealed:
+
+`vault unseal {{key-share-x}}`
+
+- Authenticate client against vault, using an authentication token:
```

Comment:
I'm not really comfortable with the wording "Authenticate client against vault". Any change we could word it some other way?

Reply:
c.f. https://github.com/tldr-pages/tldr/pull/1475#discussion_r137553819